types/google.visualization/index.d.ts

```diff
@@ -1033,6 +1033,14 @@ declare namespace google {
             draw(data: DataTable | DataView, options: PieChartOptions): void;
         }
 
+        export interface PieChartLegend {
+            alignment?: ChartLegendAlignment | undefined;
+            maxLines?: number | undefined;
+            position?: 'bottom' | 'labeled' | 'left' | 'none' | 'right' | 'top' | undefined;
+            textStyle?: ChartTextStyle | undefined;
+            numberFormat?: string | undefined;
```

Comment:
Udefined is not needed for all options.

Reply:
This was added to resolve the merge conflict created by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54321, in which all optional properties in this type definition are now ` | undefined` - for example:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/33988371025ae2e58bdaa4391d6661d4ef273242/types/google.visualization/index.d.ts#L460-L467

Reply:
cc @sandersn just checking this is the correct thing to do here?

Reply:
You are using optional chaining for that you don't need to use "undefined".

Reply:
Since typescript 4.4 there is a small difference between `axis?: string` and `axis?: string | undefined`.
With the `--exactOptionalPropertyTypes` flag, the first option allows only missing axis property, the second option also allows setting `{ axis: undefined }`.
See [exact-optional-property-types](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4-beta/#exact-optional-property-types)

[A lot](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54321) of definitely typed packages were changed to accommodate this new flag, and I think in this case it is correct to add `| undefined` because the library treats it the same way as a missing prop.